docs/webapp.md

```diff
@@ -30,11 +30,12 @@ You can change any of the settings for `sbt-node-js` and `sbt-deploy` plugins in
 enablePlugins(WebappPlugin)
 nodeProjectDir in Npm := file("clientapp")
 nodeProjectTarget in Npm := file("client-build")
-mappings in Universal += ...
+mappings in Universal +=
```

Comment:
I'd just get rid of this line as it isn't valid. The previous (with the ...) was meant to suggest you can add more to the mappings.